lib/ui/window.dart

```diff
@@ -67,6 +98,12 @@ class FlutterView {
     return platformDispatcher._viewConfigurations[viewId]!;
   }
 
+  /// The [Display] this view is drawn in.
+  Display get display {
+    assert(platformDispatcher._displays.containsKey(_viewConfiguration.displayId));
```

Comment:
Will this crash if the embedder doesn't provide display information? (I'm not sure Windows, macOS, or Linux provide display information)

Reply:
Right now they provide a display that is not quite valid. But yes, the assert would fire if the embedder failed to provide a display.

Reply:
Should we make this property nullable and force the caller to have some fallback logic if display information isn't available?

Reply:
I think we should just implement it. Making it nullable now and non-nullable later is harder once people start using it (it results in compilation errors, which make the migration in google's monorepo harder). I don't expect it to take long to implement the desktop portion of this, I just don't want to delay this patch further and it's already fairly large.

Reply:
So, what does this currently return for unimplemented platforms?

Have we verified that this is implementable on all desktop platforms we want to support? Wayland comes to mind as a platform that may not be able to tell us what display we are on (I haven't verified this, this is just from what I have heard). If that turns out to be true, what would we expect this getter to do? For that case, it may be nicer if this can just be null.

Reply:
Right now it's returning -1 for the DPR, width, and height, and a real refresh rate/display ID.

Reply:
On Linux, we can indeed get the width and height of at least the display the application is rendering to, and probably all displays. Even if Wayland doesn't have an API for this, you have to tell Wayland to connect to a particular display adapter, and it will be possible to query the information about the display size from that adapter.  GTK also has an API for this.

Reply:
Sorry, I meant GDK :) What the Linux embedding uses for windowing etc.